users/art/art.h

```diff
@@ -1,67 +1,106 @@
+// Copyright 2022 Artjoms Rizihs (@artjomsR)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
 #pragma once
 #include QMK_KEYBOARD_H
+#include "art_user_config.h"
```

Comment:
This needs to be removed. 
```suggestion
```

Reply:
Hey @drashna, sorry to necro an old thread. My bad, I actually didn't check this when you suggested it - why did you say this line needs to be removed? I just tried it and realised it doesn't compile without it: `'lang_switch_combo' undeclared (first use in this function) send_string(lang_switch_combo);` which makes sense because `lang_switch_combo` is declared in `art_user_config.h`. 

If it's no good to include .h files in user.h, then what is the alternative please?